components/rt/src/rt_estimator.m

```diff
@@ -1,35 +1,35 @@
 %% Atte Aalto
-
-% change to the root of the file
-pathToFile = fileparts(mfilename('fullpath'));
-if ~isempty(pathToFile)
-    rootFolder = [pathToFile filesep '..'];
-else
-    rootFolder = pwd();
-end
-
-cd(rootFolder);
-addpath(genpath(rootFolder));
-
-% define output folder
-outFolder = [rootFolder filesep 'output'];
-checkFolder(outFolder);
-inFolder = [rootFolder filesep 'input'];
-checkFolder(inFolder);
+% 
+% % change to the root of the file
+% pathToFile = "input/"
+% if ~isempty(pathToFile)
+%     rootFolder = [pathToFile filesep '..'];
+% else
+%     rootFolder = pwd();
+% end
+% 
+% cd(rootFolder);
+% addpath(genpath(rootFolder));
+% 
+% % define output folder
+% outFolder = [rootFolder filesep 'output'];
+% checkFolder(outFolder);
+% inFolder = [rootFolder filesep 'input'];
+% checkFolder(inFolder);
 
 %Read input data
 today = datestr(clock, 29);
-inFile = [inFolder filesep today '_clinical_monitoring_cleaned_case_and_hospital_data.xlsx'];
-if isfile(inFile)
+inFile = "input/input-data"; %[inFolder filesep today '_clinical_monitoring_cleaned_case_and_hospital_data.xlsx'];
```

Comment:
Instead of the `while True` in the Python script, the existence check can also be written there like here.

Reply:
Isn't it the same? (`if not os.path.isfile(inputfile): ` in Python checkfile)

Reply:
Sure, but not in the `src`reff_estimator.py` file, there is still the `while True` loop

Reply:
That's still not polished: I'm deleting the checks from that file now